stories/2023-02-01-Overwatch-League/OWL_story.ipynb

```diff
@@ -0,0 +1,3730 @@
+{
```

Comment:
<p>I'd be tempted to play with different colormaps, as I find it quite difficult to pick out the differences with this one (there are others e.g. <code>rocket</code>,<code>mako</code>,<code>magma</code>,<code>viridis</code> + their reverse with the <code>_r</code> suffix that use a wider range of colours whilst still being perceptually uniform).  Also, axis labels would be nice (including for the colour bar), though you do define them underneath.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>Switched to <code>viridis</code>. I like more than some of the others because it doesn't use something close to white at the extreme, which would be confusing with our use of white meaning missing.</p>